lib/presenter/Chart.js

```diff
@@ -72,7 +72,7 @@ class Chart extends Events {
             document.querySelectorAll(selectorOrEl)[0] : selectorOrEl;
 
         if (this.d3Svg) {
-            return element.parentNode.replaceChild(this.d3Svg.node(), element);
+            return this.resizeChart(selectorOrEl, true);
```

Comment:
This should fix the issue with Charts rendering with a `0` width. I'll file a ticket to investigate removing the hacks from the other existing plots.